tests/alfacase/test_migration.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from pytest_regressions.file_regression import FileRegressionFixture
+
+from alfasim_sdk._internal.alfacase.migration import migrate_alfacase_yaml_to_latest
+
+
+def test_alfacase_to_case_migration(file_regression: FileRegressionFixture) -> None:
+    """
+    Test the situation where the alfacase files have set the removed enumeration value
+    `model_default` from emulsion properties Enum, updating it to a new default value (ASIM-6137).
+    """
+
+    alfacase_file = Path(__file__).parent / "test.alfacase"
+
+    migrate_alfacase = migrate_alfacase_yaml_to_latest(
+        alfacase_file.read_text(encoding="utf-8")
+    )
+
+    file_regression.check(migrate_alfacase, extension=".alfacase")
```

Comment:
We can test the full roundtrip, so we should:

```suggestion
def test_migrate_emulsion_enums(file_regression: FileRegressionFixture) -> None:
    """
    Test the situation where the alfacase files have set the removed enumeration value
    `model_default` from emulsion properties Enum, updating it to a new default value (ASIM-6137).
    """
    case = load_case_description(DescriptionDocument.from_file(datadir / "test_migrate_emulsion_enums.alfacase"))
    assert case.physics.emulsion_relative_viscosity_model is EmulsionRelativeViscosityModelType.Brinkman1952
    assert case.physics.emulsion_droplet_size_model is EmulsionDropletSizeModelType.Brauner2001
    assert case.physics.emulsion_inversion_point_model is EmulsionInversionPointModelType.BraunerUllmann2002
```

Note:

* Moved `test.alfacase` to the data directory.
* Please remove the file generated by `file_regression`.